build-system/common/check-package-manager.js

```diff
@@ -183,9 +183,7 @@ function getNodeLatestLtsVersion(distributionsJson) {
 // If yarn is being run, perform a version check and proceed with the install.
 function checkYarnVersion() {
   const yarnVersion = getStdout(yarnExecutable + ' --version').trim();
-  const yarnInfo = getStdout(yarnExecutable + ' info --json yarn').trim();
-  const yarnInfoJson = JSON.parse(yarnInfo.split('\n')[0]); // First line
-  const stableVersion = getYarnStableVersion(yarnInfoJson);
```

Comment:
_[This comment is for line 202, not line 188]_ To make sure developers actually end up with the correct version, can we change the suggested installation command to the following?

```js
      cyan(
        `"curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version ${stableVersion}"`
      ),
```